apis/nucleus/src/components/listbox/ListBoxPopover.jsx

```diff
@@ -201,7 +204,7 @@ export default function ListBoxPopover({
               listCount={listCount}
               selections={selections}
               selectionState={selectionState}
-              keyboard={{ enabled: false }}
+              keyboard={{ innerTabStops: true, enable: false }}
```

Comment:
Do you know why it has to have these pros `{ innerTabStops: true, enable: false }`?

Reply:
Because of this: https://github.com/qlik-oss/nebula.js/blob/22c64bc0a5089a81bbf8b8d6dd06429132455095/apis/nucleus/src/components/listbox/components/ListBoxSearch.jsx#L299